### PR TITLE
Added Sequence Load information for TinyMCE extension

### LINKED ISSUE #4315: Feedback on page: /guides/v2.3/ui_comp_guide/components/wysiwyg/configure-tinymce-editor.html
# General issue
The documentation page didn‘t mention anything about adding two modules to the load order. Without adding the modules `Magento_Cms` and `Magento_PageBuilder`, it’s not possible to override the Pagebuilder default TinyMCE configuration.

## Description:

Documention lacks sufficient information to properly override the PageBuilder TinyMCE configuration.

## Possible solutions:

Please mention to add these two modules to the load order of the custom module.
```
<sequence>
    <Magento_Cms />
    <Magento_PageBuilder />
</sequence>
```

### PR DESCRIPTION
**Pull request by BarnyShergold**
## Purpose of this pull request

This pull request (PR) fixes #4315 

## Affected DevDocs pages
https://devdocs.magento.com//guides/v2.3/ui_comp_guide/components/wysiwyg/configure-tinymce-editor.html
https://devdocs.magento.com//guides/v2.4/ui_comp_guide/components/wysiwyg/configure-tinymce-editor.html

It is not necessary to include Magento_CMS in the load order as the PageBuilder module already does this.

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by rogyar** [comment]
Hi @BarnyShergold. I've removed the "Editorial" label since it's more about fixing a spelling issue or removing some pieces of the information. In this particular PR you have added a good clarification so it's definitely more than "Editorial" 🙂

**Review by rogyar — APPROVED** [review]

File: src/guides/v2.3/ui_comp_guide/components/wysiwyg/configure-tinymce-editor.md [hunk 1]
```diff
@@ -56,6 +56,13 @@ The `DefaultConfigProvider` class returns the data required to render the TinyMC
 From the example, the class `Magento\PageBuilder\Model\Wysiwyg\DefaultConfigProvider` overrides the existing configuration provided by the `Magento_CMS` module.
 
 ## Extending the TinyMCE editor
+When extending the TinyMCE editor it is necessary to add the PageBuilder module to the load order of the custom module by adding the following to the `module.xml` file.
```
Comment (dobooth) [inline src/guides/v2.3/ui_comp_guide/components/wysiwyg/configure-tinymce-editor.md:59]: Blank line between header and content.

**Review by dobooth — CHANGES_REQUESTED** [review]

File: src/guides/v2.3/ui_comp_guide/components/wysiwyg/configure-tinymce-editor.md (hunk 1 above)
Comment (dobooth) [reply src/guides/v2.3/ui_comp_guide/components/wysiwyg/configure-tinymce-editor.md:59]: Wonder if the linter should have caught that.

Comment (BarnyShergold) [reply src/guides/v2.3/ui_comp_guide/components/wysiwyg/configure-tinymce-editor.md:59]: Sorry I'm not seeing what you mean. Do you mean line 60?

All other sections have a blank line between content and ```xml

Comment (dobooth) [reply src/guides/v2.3/ui_comp_guide/components/wysiwyg/configure-tinymce-editor.md:59]: Line 59 should be a blank line. "When extenting..." is line 60 and then yes, another blank line between the the content and the code block.

Comment (BarnyShergold) [reply src/guides/v2.3/ui_comp_guide/components/wysiwyg/configure-tinymce-editor.md:59]: Ah, you mean ADD a blank line 😄 
Gotta be more precise guv 😉

**Review by dobooth — APPROVED** [review]

**Pull request merged**